salt/config/__init__.py

```diff
@@ -956,7 +956,7 @@ def _gather_buffer_space():
     # The size of key that should be generated when creating new keys
     'keysize': int,
 
-    # The transport system for this daemon. (i.e. zeromq, raet, etc)
+    # The transport system for this daemon. (i.e. zeromq, tcp, etc)
```

Comment:
Isn't it better to specify all 3 currently acceptable values here? `(i.e. zeromq, tcp or detect)`

Reply:
Yes, probably. Though I believe the `detect` functionality is broken. :(

I will fix that comment though. :D